packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -3967,4 +3968,44 @@ void main() {
   },
     variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }),
   );
+
+  testWidgets('Selection of Latin and non-Latin texts.', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const MaterialApp(
+          home: Material(
+            child: SelectableText(
+              'Latin 中文(non-Latin)',
+              selectionWidthStyle: ui.BoxWidthStyle.tight,
+              selectionHeightStyle: ui.BoxHeightStyle.max,
+            ),
+          ),
+        ),
+      );
+
+      // Long press at 'e' in 'def'.
+      final Offset ePos = textOffsetToPosition(tester, 5);
```

Comment:
This test does not seem to test the right thing, you will probably need a golden test. See https://github.com/flutter/flutter/blob/77b4505c8014330fba2f2706c658a64bc5d1891b/packages/flutter/test/material/text_field_test.dart#L511